ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/service/ViewSpecificationConverter.java

```diff
@@ -48,72 +60,90 @@ public static Model modelFromView(ViewSpecification view) {
 		return model;
 	}
 
-	private static ViewName viewNameFromStatements(List<Statement> statements, String collectionName) {
+	public Resource getIRIFromViewName(ViewName viewName) {
+		return createResource(hostname + "/" + viewName.getCollectionName() + "/" + viewName.getViewName());
+	}
+
+	private ViewName viewNameFromStatements(List<Statement> statements, String collectionName) {
 		String nameString = statements.stream()
 				.filter(statement -> statement.getPredicate().toString().equals(TREE_VIEW_DESCRIPTION))
-				.map(statement -> statement.getSubject().toString()).findFirst()
+				.map(statement -> statement.getSubject().getLocalName()).findFirst()
 				.orElseThrow(() -> new ModelToViewConverterException("Missing type: " + TREE_VIEW_DESCRIPTION));
 
 		return new ViewName(collectionName, nameString);
 	}
 
-	private static List<RetentionConfig> retentionListFromStatements(List<Statement> statements) {
+	private List<RetentionConfig> retentionListFromStatements(List<Statement> statements) {
 		List<RetentionConfig> retentionList = new ArrayList<>();
 		for (Resource retention : statements.stream()
 				.filter(new ConfigFilterPredicate(RETENTION_TYPE))
 				.map(Statement::getSubject).toList()) {
 			List<Statement> retentionStatements = retrieveAllStatements(retention, statements);
 			RetentionConfig config = new RetentionConfig();
-			config.setName(retention.toString());
-			config.setConfig(extractConfigMap(retentionStatements));
+			Map<String, String> configMap = extractConfigMap(retentionStatements);
+			// TODO verify Retention name corresponds with a valid policy
```

Comment:
When will this be done? If it's part of a story, let's add the ticket number in the TODO.. makes it easier to later cleanup TODOs